remarkable-review/The End of Paper.md

```diff
@@ -0,0 +1,101 @@
+
+How was I convinced to replace a decade-old allegiance to analog in under ten minutes? Well, let's just say the reMarkable tablet is aptly named.
+
+I'll walk you through my paper-based workflows, the various digital products I've tried to replace them with, and then my reMarkable experience.
+
+## My love affair with paper
+<photo of my stacks of moleskines>
+The first dated entry I could find from one of my notebooks is X/X/2000. The writings began as the passionate overwrought ramblings of a high schooler. During [undergrad](http://cs.trinity.edu/) and [grad school](https://cs.unc.edu/), I wrote haphazardly, not really establishing a structured habit for my usage of paper.
+
+Beginning in 2007, I got serious. I carried a paper notebook with me everywhere. And I mean everywhere. I started my day by cracking it open, carried it to all my meetings, lunches, church, travel, everywhere. Notebook-carrying became such a fixture of my identity that when I got married in 2010 the write-everything-down habit was hilariously caricatured by a friend using a 3 foot by 4 foot cardboard notebook, acting out me writing down every word while on a date with my wife.
+
+<photo of Eric with a "paper notebook">
+
+I wrote nearly every day. I also evolved my system continually. I moved from a single notebook for capturing everything to two: one for a daily working log and one for personal, long-form journaling. I experimented with different formatting techniques for pages. I dabbled in a variety of paper todo systems. I experimented with different notebook form factors: soft cover, hard cover, small, medium, large, lined, unlined, grid. I scribbled my way through sheafs of dead tree matter. Writing even seem to reshape my brain: my habit of thought defaulted to externalizing ideas on paper or whiteboard.
+
+And I loved it.
+
+I loved starting a new day with a fresh page, noting the day's date, and opening myself to the possibilities in front of me. I loved dedicating time to look back on life, sorting through emotions, churning thought-trains, and half-made plans. From brain to hand to pen, the ink flowing onto the page created a black-and-beige clarity that seemed more difficult when staring at a glowing rectangle or typing on my keyboard.
+
+All that's to say, I was and am still deeply in love with paper. 
+
+## A dalliance with digital
+As wonderful as paper had been to me, as a digital native, I was constantly aware of the tradeoff I was making: digital has its advantages.
+
+I got my first job as a software developer in 2003. Which means I type a lot. My keyboard is often the most expedient tool for making quick notes.
+
+It began with simple "scratch notes", outlining how I was planning to tackle a problem or something I wanted to remember for later in the day. Eventually, I found myself needing to recall something from a previous day or share something with a colleague.
```

Comment:
The idea in the last sentence deserves something more to link it to your pen/paper evolution, e.g. "But as with my pen and paper note-taking, my use cases for digital notes continued to grow. Again, I had to create a system to facilitate my thought process."

Reply:
I see now that you get here in the next paragraph, but I think your sentence here about eventually needing to recall something from a previous day is where the story of the evolution begins. Not sure what the solution is but this and the next paragraph could use some more tinkering.